ext/pycall/pycall.c

```diff
@@ -1056,6 +1062,7 @@ pycall_pyobject_wrapper_wrapper_method(int argc, VALUE *argv, VALUE wrapper)
       pycall_Py_DecRef(attr);
       pycall_pyerror_fetch_and_raise("PyObject_SetAttrString in pycall_pyobject_wrapper_wrapper_method");
     }
+    pycall_Py_DecRef(attr);
```

Comment:
Could you please move this line after the call of `PyObject_SetAttrString` above, and remove the call of `pycall_Py_DecRef` in `if (res == -1)` block?

Reply:
I see. I will move this, followings and other `DecRef`s just after the calls and delete useless `DecRef`s in `if (res == -1)` block.